compiler/src/parsing/lexer.re

```diff
@@ -249,6 +249,11 @@ let rec token = lexbuf => {
   | "try" => positioned(TRY)
   | "throw" => positioned(THROW)
   | "catch" => positioned(CATCH)
+  | "async" => positioned(ASYNC)
+  | "await" => positioned(AWAIT)
```

Comment:
@phated doesn't want to reserve these two, so let's remove them.

Reply:
Is there explicit reason why though? Perhaps it may come in handy in the future-

Reply:
Don't want to lock us into a system we might not use.